scripts/lib/libvirt/admin-config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import argparse
+from string import Template
+import sys
+
+import libvirt
+import libvirt_helpers
+
+conn = libvirt.open("qemu:///system")
+if not conn:
+    print("Failed to open connection to the hypervisor")
+    sys.exit(1)
+
+parser = argparse.ArgumentParser(description="Create Admin Node Config.")
+parser.add_argument("cloud", type=str, help="Name of the cloud")
+parser.add_argument("adminnodememory", type=str, help="Amount of Memory")
+parser.add_argument("adminvcpus", type=str, help="Number of VCPUs")
+parser.add_argument("emulator", type=str, help="Emulator")
+parser.add_argument("adminnodedisk", type=str, help="Disk of the Admin Node")
+parser.add_argument("localrepositorymount", type=str,
+                       help="Local Repository Mount XML")
+
+args = parser.parse_args()
+
+
+def main():
+    cloud = args.cloud
```

Comment:
> But if the plan is continue to extend the replacement in favor of Python, @aspiers is right.

That direction brings a lot of benefits - modular testable code without lots of `fork()`s and lots of CLI argument parsing.

But even if we don't go that direction, the current `if __name__ == "__main__":` is simply wrong.  Currently if you import the file, it will attempt to run the arg parsing code and nothing else.